src/Containers/Article/Single/singleArticle.jsx

```diff
@@ -62,13 +77,180 @@ export class SingleArticle extends Component {
     );
   };
 
+  commentBlock = () => {
+    const {
+      isAuth,
+      article: { loading, comments = [] },
+      username: currentUsername,
+    } = this.props;
+
+    const { body } = this.state;
+
+    /**
+     * Toggles the comment textarea
+     *
+     * @param {*} index
+     * @param {*} status
+     * @returns {void}
+     */
+    const toggler = index => {
+      this.setState(prevState => ({ [index]: !prevState[index] }));
+    };
+
+    return (
+      <React.Fragment>
+        <div className="form comment-block">
+          <div className="comment-label">Comments</div>
+          <div className="wrapper" style={{ display: !isAuth ? 'none' : null }}>
+            <textarea
+              name="comment"
+              cols="30"
+              rows="10"
+              value={body}
+              onChange={e => this.setState({ body: e.target.value })}
+            />
+            <Button text="POST" onClick={() => this.handlePostComment()} loading={loading} />
+          </div>
+        </div>
+        {comments.map(({ body: text, favoritesCount, User: { username, image }, id }, index) => {
+          const { state } = this;
+          const { updateBody } = state;
+
+          return (
+            <div className="comment-card" key={index}>
+              <div className="author-details-column">
+                <img src={image || defaultProfile} alt="" />
+                <div className="names">
+                  <h4>{username}</h4>
+                </div>
+              </div>
+              <div className="comments">
+                <span>{text}</span>
+
+                <div className="option">
+                  <Icon icon={faThumbsUp} />
+                  <span className="">{favoritesCount}</span>
+                  {currentUsername === username ? (
```

Comment:
You can use `&&` here instead of a ternary operator you will have a short and clean syntax

Reply:
@dusmel Alright